pubkey.go

```diff
@@ -79,18 +79,13 @@ func (p *PubKey) load() {
 
 // Fetch user's public key data from GitHub
 func (p *PubKey) FillKeys() *PubKey {
+	cli := github.NewClient(nil)
 	for u, _ := range p.users {
-		cli := &http.Client{Transport: p.transport}
```

Comment:
I will be happy if HTTP transport setting is removed! 👍

Reply:
Thanks. I'm going to remove it tonight.